src/main/java/be/bendem/bukkit/orebroadcast/handlers/PluginMessage.java

```diff
@@ -0,0 +1,90 @@
+package be.bendem.bukkit.orebroadcast.handlers;
+
+import be.bendem.bukkit.orebroadcast.OreBroadcast;
+
+import com.google.common.io.ByteArrayDataInput;
+import com.google.common.io.ByteArrayDataOutput;
+import com.google.common.io.ByteStreams;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.messaging.PluginMessageListener;
+
+import java.io.*;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PluginMessage implements PluginMessageListener {
+    private OreBroadcast plugin;
+
+    public PluginMessage(OreBroadcast plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public void onPluginMessageReceived(String channel, Player player, byte[] message) {
+        if(!channel.equals("BungeeCord"))
```

Comment:
espace entre le if et la (
```suggestion
        if (!channel.equals("BungeeCord"))
```